src/PolyhedralGeometry/Polyhedron/properties.jl

```diff
@@ -397,7 +400,7 @@ julia> volume(C)
 4
 ```
 """
-volume(P::Polyhedron) = (pm_object(P)).VOLUME
+volume(P::Polyhedron) = (pm_object(P)).VOLUME::Int
```

Comment:
In most cases this will be a polymake rational, but it might also return other types where there is no mapping yet, those will be hidden in a C++ `PropertyValue` type:
```julia
julia> volume(archimedean_solid("icosidodecahedron"))
PropertyValue wrapping pm::QuadraticExtension<pm::Rational>
15/2+17/6r5
```
So I think we should leave this out for now.
```suggestion
volume(P::Polyhedron) = (pm_object(P)).VOLUME
```